components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx

```diff
@@ -53,7 +54,7 @@ const ActiveDeposits: FC<ADProps> = ({ parentHeight, pools }: ADProps) => {
                                 onMouseLeave={() => setShowButton(-1)}
                             >
                                 <TableCell>{pool.market}</TableCell>
-                                <TableCell>{toPercent(pool.apy.toNumber())}</TableCell>
+                                <TableCell>{toPercent((pool.apy ?? new BigNumber(0)).toNumber())}</TableCell>
```

Comment:
```suggestion
                                <TableCell>{toPercent((pool?.apy ?? new BigNumber(0)).toNumber())}</TableCell>
```
If pool.apy works then you shouldnt need the ?? new BigNumber(0) I think pool.apy has a default 0 value

Reply:
updated to just `toPercent(pool.apy?.toNumber())`